README.md

```diff
@@ -1,3 +1,7 @@
 # user-agents
 
 Occasionally fetches the latest user agent strings for major browsers and OSs.
+
+## Contributing
+
+After any changes format your code using the [Black](https://pypi.org/project/black/) formatter with the default args.
```

Comment:
Should it be added to `requirements.txt`?

Reply:
Added